packages/components/src/components/text/text.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+import { isEmptyObject } from '@deriv/shared';
+import { TGenericStringValueObject } from '../types';
+
+interface ITextProps extends HTMLProps<HTMLElement> {
+    align?: string;
+    weight?: string;
+    line_height?: string;
+    styles?: TGenericStringValueObject;
```

Comment:
@jim-deriv what do you think about this one? working for me 🙂:
```suggestion
    styles?: React.CSSProperties;
```